internal/cmd/logout_test.go

```diff
@@ -7,8 +7,10 @@ import (
 	"path/filepath"
 	"sync/atomic"
 	"testing"
+	"time"
 
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/infrahq/infra/api"
```

Comment:
First party imports should be in its own section after the 3rd party imports

Reply:
fixed!